addons/actions.md

```diff
@@ -95,6 +95,26 @@ The Timer object supports the following methods:
 * `hasTerminated`: returns true if the code has run and completed
 * `reschedule(AbstractInstant instant)`: reschedules the timer to execute at the new time. If the Timer has terminated this method does nothing.
 
+### Thing Status Action
+
+`getThingStatusInfo(String thingUid)`: Gets ThingStatusInfo of the given thing identified by thingUid and returns the result as ThingStatusInfo
```

Comment:
Explaining `getThingStatusInfo` with "Gets ThingStatusInfo" doesn't do much good. I caught myself looking at the example below to understand what this section is all about. Maybe you could condense this sentence and the following paragraph.